Remove the declaration of group_member since it has already been
declared in unistd.h

Signed-off-by: Dongxiao Xu <dongxiao.xu@intel.com>

Upstream-Status: Inappropriate [embedded specific]

diff -ruN which-2.20-orig/bash.c which-2.20/bash.c
--- which-2.20-orig/bash.c	2010-08-03 12:59:08.897088878 +0800
+++ which-2.20/bash.c	2010-08-03 13:17:23.406097212 +0800
@@ -46,7 +46,6 @@
  * - changed all occurences of 'gid_t' into 'GID_T'.
  * - exported functions needed in which.c
  */
-static int group_member (GID_T gid);
 static char* extract_colon_unit (char const* string, int* p_index);
 
 /*===========================================================================
